ide-migration/server/migration/api/migration-service.js

```diff
@@ -365,6 +354,48 @@ class MigrationService {
         hdiFile.setText(hdiJson);
     }
 
+    addFilesWithoutGenerated(userData, workspace, locals) {
+        for (const local of locals) {
```

Comment:
Could you rename `locals` to `localFiles` and `local` to `localFile`. Same goes for other loops

Reply:
+1